source/clear-linux/guides/maintenance/ister.rst

```diff
@@ -0,0 +1,89 @@
+.. _ister:
+
+ister.py
+########
+
+The `ister.py tool`_ is a template based installer used by |CL-ATTR| to produce
+images for each release. The same ister tool is available for use in |CL| to
+create custom images based on an upstream image.
+
+Description
+***********
+
+|CL| is a rolling release and produces on average 10 releases per week using the
+ister tool. With each release we produce multiple
+`image types for different environments`_  and use cases such as installers,
+Hyper-V, KVM, or VMWare.
+
+Each image has a JSON configuration file used by ister to generate the image.
+These JSON configuration files describe the image type, partitions, version,
+and which bundles will be preinstalled by default with the image. For each image
+type we produce, the corresponding JSON configuration file for the image is also
+published.
+
+The :ref:`mixer<mixer>` tool also uses ister to build images for your custom
+mix. Like upstream images, a JSON configuration file is defined for the image,
+which ister uses to generate the image. Refer to the :ref:`mixer<mixer>` guide
+for instruction on using ister to build an image for a custom mix.
+
+Example: Recreate an upstream image
+***********************************
+
+The published configuration files for upstream images may be used to recreate an
+image, for example when you want to:
+
+* Use an older version of |CL| and the image is no longer available (only after
+  March 2017).
+* Customize the partitions of an image.
+* Customize the bundles preinstalled in an image.
+* Run your own post installation script.
+
+Prerequisites
+=============
+
+#. |CL| installed.
+
+#. :command:`os-installer` bundle is installed.
+
+#. ister.py has sudo privilege.
+
+Recreate upstream image
+=======================
+
+Follow these steps to recreate an upstream image based on the image's JSON
+configuration file:
+
+#. Download the JSON configuration file for the desired image:
+
+   * `Configuration files for the current release`_
+   * `Previous releases`_ (only after March 2017)
+
+   For a previous release, the configuration file for the specific release is
+   located at
+   ``https://cdn.download.clearlinux.org/releases/<version number>/clear/config/image/``

```

Comment:
Will you please add a note? Use something like: 
Replace `<version number>` with your preferred CL image.

My fear here is that users will cut and paste this, get a 404, and be disappointed. A simple note will help.